### ISSUE TITLE
Excess padding when using `<img slot="logo" ...>`

### ISSUE THREAD

**Issue by ngnpope** [mrin9/RapiDoc#576]
When overriding the logo using slots some extra padding is applied:

![Screenshot from 2021-10-01 16-38-46](https://user-images.githubusercontent.com/2855582/135648477-e4053494-c64d-43f3-b818-a5816e8e2779.png)

In many cases this seems to make things look incorrect. It appears to be caused by the following:

```css
::slotted(*){
  padding:16px 30px 0 16px;
}
```

It feels like this ought to be removed and that padding/margins should be left up to the user.

**Comment by ngnpope** [comment]
I worked around this with `*[slot] { padding: 0; }`.

**Comment by mrin9** [comment]
padding provided is just a default, you can always override it with your own 

```html
<rapi-doc spec-url="some.yaml" render-style = "read">
  <img slot="nav-logo" src = "../images/dog.png" style="padding:0px"> 
</rapi-doc>
```
were you not able to do this ?

**Comment by ngnpope** [comment]
Yes, it just seemed like a strange default, especially given that it doesn't affect the default logo.

**Comment by mrin9** [comment]
there are many places you will find these sort of strange defaults that we have applied to reduce our support calls.

For instance, in this case many of our users such as tech-writers who arnt good with css styling would like to provide an image and a spec and be done with. however, they wud later create a support ticket on how to provide spacing around the image as it looks too crammed up.  

But this default was intended for the logo on the navigation bar, from the image you provided, it seems like got applied to header logo as well, which is an oversight and I will fix it

**Comment by ngnpope** [comment]
Yes, sorry for not being clear. It is specifically the logo in the header, not the nav bar. Thanks!
